README.md

```diff
@@ -4,23 +4,31 @@
 [![License](https://img.shields.io/badge/license-Apache--2.0-green.svg)](LICENSE.md)
 ![Unity](https://img.shields.io/badge/unity-2020.2+-brightgreen)
 
+---
+
+We're currently working on lots of things! Please take a short moment fill out our [survey](https://unitysoftware.co1.qualtrics.com/jfe/form/SV_0ojVkDVW0nNrHkW) to help us identify what products and packages to build next.
+
+---
+
+## Introduction
+
 URDF Importer allows you to import a robot defined in [URDF](http://wiki.ros.org/urdf/XML) format in a Unity scene. URDF defines the geometry, visual meshes, kinematic and dynamic attributes of a Robot. Importer parses a URDF file and imports it into Unity using PhyX 4.0 articulation bodies.
 
-# Using the Package
+## Using the Package
 
-## Adding the URDF package
+### Adding the URDF package
 
 1. Open the Package Manager from Unity Menu. Click `Window -> Package Manager`. A new package manager window will appear.
 
 2. Click on the `+` sign on the top left corner of the package manager window and click on `Add Package from Git URL`. 
 
 <img src = "images~/Package_manager_add.png">
 
-3. Enter the git URL for the URDF Importer with the latest version tag (currently v0.5.0) `https://github.com/Unity-Technologies/URDF-Importer.git?path=/com.unity.robotics.urdf-importer#v0.5.0` in the text box and press `Enter`.
+3. Enter the git URL for the URDF Importer with the latest version tag (currently v0.5.1) `https://github.com/Unity-Technologies/URDF-Importer.git?path=/com.unity.robotics.urdf-importer#v0.5.1` in the text box and press `Enter`.
```

Comment:
sorry, just noticed this! is the package version 0.5.2?

Reply:
Good catch! 
https://github.com/Unity-Technologies/URDF-Importer/pull/177